### ISSUE TITLE
Extra lines, and "Warning: the return type should be provided." when trying numpydoc as output.

### ISSUE THREAD

**Issue by masasin** [dadadel/pyment#16]
`test.py` contains docstring in the google format:

```
def test(food, spam, water, n=4):
    """A short description.

    Some more descriptions here.

    Args:
        food: Something to eat.
        spam (string): A type of food.
        water (list, tuple): Multiple types.
        n (int): The number of things. Default is 4.

    Returns:
        The number of things.
    """
```

When trying to convert to numpydoc, I used: `pyment test.py -i "google" -o "numpydoc". It created a patch file, and gave me a warning saying: `Warning: the return type should be provided.`

The patch file also contained some errors. These are the contents of `test.py.patch`.

```
# Patch generated by Pyment v0.3.1

--- a/test.py
+++ b/test.py
@@ -1,14 +1,28 @@
 def test(food, spam, water, n=4):
     """A short description.
-
+    
     Some more descriptions here.
 
-    Args:
-        food: Something to eat.
-        spam (string): A type of food.
-        water (list, tuple): Multiple types.
-        n (int): The number of things. Default is 4.
+    Parameters
+    ----------
+    food :
+        Something to eat.
+    spam  : string
+        A type of food.
+    water  : list
+        Multiple types.
+    n  : int
+        The number of things. Default is 4.
+    spam :
+        
+    water :
+        
+    n :
+         (Default value = 4)
 
-    Returns:
+    Returns
+    -------
+    
         The number of things.
+
     """
```

This has some errors:

* There are extra, separate lines for all items that have a type. (The order does not matter, and they are added at the end.)
* Even though it was specified, it added a `(Default value = 4)` to `n`.
* It added an extra line after the underline for `Returns`

The first two output issues occurred with `reST` as the output parameter as well, but there was no warning. The `reST` patch:

```
# Patch generated by Pyment v0.3.1

--- a/test.py
+++ b/test.py
@@ -1,14 +1,18 @@
 def test(food, spam, water, n=4):
     """A short description.
-
+    
     Some more descriptions here.
 
-    Args:
-        food: Something to eat.
-        spam (string): A type of food.
-        water (list, tuple): Multiple types.
-        n (int): The number of things. Default is 4.
+    :param food: Something to eat.
+    :param spam : A type of food.
+    :type spam : string
+    :param water : Multiple types.
+    :type water : list
+    :param n : The number of things. Default is 4.
+    :type n : int
+    :param spam: 
+    :param water: 
+    :param n:  (Default value = 4)
+    :returns: The number of things.
 
-    Returns:
-        The number of things.
     """

```

**Comment by dadadel** [comment]
Hi @masasin,

Concerning the Warning, it is due to the fact that in the Numpydoc documentation I found only examples of Return section providing return types. I think it is useless as it can be written without any rtype (and it is the case). So I will remove this warning.

Concerning the errors, it is due to the fact that I didn't expect having a space character between the parameter name and the parenthesis. Thus if you write `spam(string)`, `water(list, tuple)`,... it will work fine.
I will fix this quickly!

**Comment by dadadel** [comment]
It is now fixed in commit 7b4fd871a0f30316149d709e419babfbf7049c74 (in v0.3.2)
